src/app/store.ts

```diff
@@ -1,10 +1,12 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { currentTodoSlice } from '../features/currentTodo';
+import { filterSlice } from '../features/filter';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(todosSlice, currentTodoSlice, filterSlice);
```

Comment:
Since 'combineSlices' is not a valid function, replace it with 'combineReducers' to properly combine the reducers from the slices.